File: Angular/client-app/src/app/models/vehicle.ts
import { Observable } from "rxjs/internal/Observable";
import { TypeOfVehicle } from "src/app/models/typeOfVehicle";
import { Service } from "src/app/models/service";

export class Vehicle {
    id: number;
    Model: string;
    Manufactor: string;
    Year: number;
    Description: string;
    PricePerHour: number;
    Unavailable: boolean;
    Images: string[];
    Type: TypeOfVehicle;
    Service: Service;

    constructor(id:number, Model:string,Manufactor: string, Year: number, Description: string, PricePerHour: number, Unavailable: boolean, Images: string[], Type: TypeOfVehicle, Service: Service)
    {
        this.id = id;
        this.Model= Model;
        this.Manufactor = Manufactor;
        this.Year = Year;
        this.Description = Description;
        this.PricePerHour = PricePerHour;
        this.Unavailable = Unavailable;
        this.Images = Images;
        this.Type = Type;
        this.Service = Service;
    }

}

File: RentApp/Models/Entities/Vehicle.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace RentApp.Models.Entities
{
    public class Vehicle
    {
        public int Id { get; set; }
        public string Model { get; set; }
        public string Manufactor { get; set; }
        public int Year { get; set; }
        public string Description { get; set; }
        public decimal PricePerHour { get; set; }
        public bool Unavailable { get; set; }
        public string VehicleImagesBase { get; set; }
        public virtual List<string> Images { get; set; }
        public virtual TypeOfVehicle Type { get; set; }
    }
}